tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/EntityRecognizerRecognizerTests.cs

```diff
@@ -1,8 +1,11 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading;
```

Comment:
Yeah these 2 test classes for sure made me do a double take when I first encountered them, just from their names.

However digging into it further, they're separating `RecognizeEntitiesAsync` vs. `RecognizeAsync`
- `RecognizeAsync` for a lot of classes calls `RecognizeEntitiesAsync` from within its body, so this is where the duplication comes in
- However I'm not so sure about removing this duplication, because we want to test that Entities are returned/recognized properly from within the context of `RecognizeAsync` as well as just standalone